CHANGELOG.md

```diff
@@ -12,10 +12,13 @@ the code was deployed.
 ## [Unreleased]
 
```

Comment:
Reminder to self that upgrading `brave-alert-lib` will turn off the timestamps in `helpers.log`, so will need to add them back in in Kubernetes.

Reply:
do you want to make a readme update here for including --timestamps?

Reply:
Done.